sites/fast-tooling-examples/app/style.css

```diff
@@ -1,3 +1,63 @@
+/* This prevents a flash of unstyled components */
+:not(:defined) {
+    visibility: hidden;
+}
+
+:root {
+    --badge-fill-primary: #FB356D;
+    --badge-color-text: white;
+}
+
 body {
     margin: 0;
-}
\ No newline at end of file
+}
+
+.main {
+    display: grid;
+    grid-template-columns: 240px auto;
+    grid-template-rows: 44px auto;
+    min-height: 100vh;
+}
+
+.toolbar {
+    grid-column-start: 1;
+    grid-column-end: 3;
+    grid-row-start: 1;
+    grid-row-end: 1;
+    display: grid;
+    grid-template-columns: 50% 50%;
+    border-bottom: 1px solid #e1e1e1;
```

Comment:
No design system values?

Reply:
A design will be worked out at a later date, this site has no schedule to be live so design details are kept to a minimum.